<?xml version="1.0"?>
<launch>
	<group ns="intaking">
		<rosparam command="load" file="$(find behaviors)/config/2025_intaking_config.yaml"/>
		<node name="intaking_server_2025" pkg="behaviors" type="2025_intaking_server.py" output="screen"/>
	</group>
</launch>